### PR TITLE
DHFPROD-6610: Move the DB selection to the top in Explorer home page

### PR DESCRIPTION
**Pull request by timur-isangulov**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- N/A Added Tests
  

- ##### Reviewer:

- N/A Reviewed Tests

### CODE REVIEW

**Review by jyangML — APPROVED** [review]

File: marklogic-data-hub-central/ui/src/components/zero-state-explorer/zero-state-explorer.tsx [hunk 1]
```diff
@@ -166,7 +150,26 @@ const ZeroStateExplorer = (props) => {
...
                     </div>
                   </Col>
-                  <Col span={12}>
+                </Row>
+                <br />
+                <Row>
+                  <Col span={24}>
+                    <div className={styles.input}>
+                      <Input
+                        style={{width: 700}}
```
Comment (xnikhil08) [inline marklogic-data-hub-central/ui/src/components/zero-state-explorer/zero-state-explorer.tsx:159]: Can we move the css styles to the associated module.scss file for this component ?

File: marklogic-data-hub-central/ui/src/components/zero-state-explorer/zero-state-explorer.tsx [hunk 2]
```diff
@@ -130,26 +130,10 @@ const ZeroStateExplorer = (props) => {
...
-                  </Col>
-                </Row>
-                <Row>
-                  <br />
-                  <Col span={12}>
                     <div className={styles.database}>
                       <p className={styles.databaseLabel}>Database:</p>
                       <MLRadio.MLGroup
-                        style={{}}
+                        style={{width: "160px"}}
```
Comment (xnikhil08) [inline marklogic-data-hub-central/ui/src/components/zero-state-explorer/zero-state-explorer.tsx:136]: Same comment here as well, let's see if you can use className attribute here and place the css to the modiule.scss file.

**Review by xnikhil08 — CHANGES_REQUESTED** [review]

**Review by wooldridge — APPROVED** [review]
Looks good.

DHFPROD-6610

I also see the following fixed in this branch, no more crashing when query name is edited: https://project.marklogic.com/jira/browse/DHFPROD-6623

File: marklogic-data-hub-central/ui/src/components/zero-state-explorer/zero-state-explorer.tsx (hunk 1 above)
Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/src/components/zero-state-explorer/zero-state-explorer.tsx]: Sure, I just pushed commit. Can you please review again.

File: marklogic-data-hub-central/ui/src/components/zero-state-explorer/zero-state-explorer.tsx (hunk 2 above)
Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/src/components/zero-state-explorer/zero-state-explorer.tsx]: pushed commit.

**Review by xnikhil08 — APPROVED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Pull request merged**